topics/tutorials/postfix_completion/postfix_templates.md

```diff
@@ -0,0 +1,77 @@
+[//]: # (title: Postfix Templates Implementation)
```

Comment:
the page title seems a bit long. not sure. mb "Basic Postfix Templates"

Reply:
I had "Providing Postfix Templates" originally, but someone edited it.

If the current one isn't good, I would go with simple "Postfix Templates" (and "Advanced Postfix Templates" as a continuation), not the "Basic Postfix Templates".

Reply:
Changed to "Postfix Templates".